io.github.shiftey.Desktop.metainfo.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
FTR this should be maintained upstream and not here.

Reply:
I'd love to understand more about this, as we're using GitHub releases for some of this metadata so I'm not sure about putting that back into version control there...